### PR TITLE
Ignore files when exporting package

### PR DESCRIPTION
**Pull request by simonschaufi**
This commit is part of a campaign to reduce the amount of data transferred to save global bandwidth and reduce the amount of CO2. See https://github.com/Codeception/Codeception/pull/5527 for more info.

### CODE REVIEW

**Comment by simonschaufi** [comment]
@aik099 ping

**Comment by aik099** [comment]
Merging, thanks @simonschaufi .

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/tests export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis export-ignore
```
Comment (stof) [inline .gitattributes:4]: wrong file name. The file is named `.travis.yml`

Comment (stof) [reply .gitattributes:4]: @simonschaufi it looks like you have this mistake in other PRs you sent, btw

Comment (simonschaufi) [reply .gitattributes:4]: oh yes, right.

**Pull request merged**